viz_scripts/docker/start_notebook.sh

```diff
@@ -5,9 +5,8 @@
 echo "DB host = "${DB_HOST}
 if [ -z ${DB_HOST} ] ; then
     local_host=`hostname -i`
-    sed "s-localhost-${local_host}_" conf/storage/db.conf.sample > conf/storage/db.conf
-else
-    sed "s-localhost-${DB_HOST}-" conf/storage/db.conf.sample > conf/storage/db.conf
+    export DB_HOST=$local_host
+    echo "Setting db host environment variable to localhost"
```

Comment:
nit, future fix: this can be removed and simplified (similar to https://github.com/e-mission/e-mission-server/pull/961#discussion_r1605946966)